.github/workflows/ci.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
     needs:
       - lint
       - unit-test
-    uses: canonical/data-platform-workflows/.github/workflows/build_charms_with_cache.yaml@v2
+    uses: canonical/data-platform-workflows/.github/workflows/build_charms_with_cache.yaml@v2.3.9
```

Comment:
This makes sure we're building charms with charmcraft 2.2, since 2.3 fails to pass the charm_helper lib.